10-std.tex

```diff
@@ -368,6 +368,98 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
+\section{Template}
+\label{std-template}
+
+Haxe comes up with a standard Template System with an easy to use syntax which is interpreted by a lightweight class called \type{haxe.Template}.
```

Comment:
Lightweight is okay there, it carries the information that the implementation is quite basic and people might want to consider a different/real template implementation for more serious stuff.